cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -242,7 +242,11 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         callAnalytics: function(event) {
             event.preventDefault();
             window.analytics.track('edx.bi.highlights.' + event.target.innerText.toLowerCase());
-            this.save(event);
+            if (event.target.className.indexOf('save') !== -1) {
```

Comment:
Can we replace the -1 with a variable?

Reply:
I would argue that this is a JavaScript idiom and making a variable for -1 would be like making a variable for None in python. Just seems weird to me. If we were using ES6 I could use the `contains()` function which doesn't involve -1, but alas...